### ISSUE TITLE
Create test course & drive with joystick 

### ISSUE THREAD

**Issue by JonasPetersenFAU** [amosproj/amos2022ss03-turtlebot-fleet-management#6]
## User story
1. As a team
2. We need to get used to the robot and try the navigation tools 
3. So that we can try out the robot and get a first hands-on exposure.

## Acceptance criteria
* One asy lap with the robot
* Have got familiar with joystick
* Created a test course

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by nmarkert** [comment]
### How to Run
First run
`ros2 launch turtlebot2_ros2 bringup.launch.py`

In a second terminal run
`ros2 launch teleop_twist_joy teleop-launch.py joy_config:='xbox'`
Now you are able to move the Turtlebot with the Controller by holding the RB-Button and using the Left Joystick.

If you want to control the Turtlebot with the Keyboard, you can instead run
`rosrun teleop_twist_keyboard teleop_twist_keyboard.py`

### Sources
 - https://github.com/wn1980/turtlebot2_ros2
 - https://github.com/ros2/teleop_twist_joy
 - https://github.com/ros2/teleop_twist_keyboard

**Comment by Sebastian2023** [comment]
https://user-images.githubusercontent.com/92729241/169974523-7800ca45-1cb6-4db1-9e06-02ead1980533.mp4
